MAINTAINER.md

```diff
@@ -12,6 +12,46 @@
 - Julien Salleyron [@juliens](https://github.com/juliens)
 - Nicolas Mengin [@nmengin](https://github.com/nmengin)
 
+
+## PR review process:
+
+* If needed, a specific maintainer can be required to make a review (according to its knowledge areas)
```

Comment:
I'd rephrase this to

_If needed, a specific maintainer familiar with a particular domain can be requested for the review._

I'd also move the remark after the next block (i.e., between lines 21 and 23) since it's more of an add-on information to the more central description of how many LGTMs are needed.